ui/js/controllers/perf/alerts.js

```diff
@@ -67,7 +70,39 @@ perf.controller(
                 }
             });
         }]);
+perf.controller(
+    'EditAlertSummaryNotesCtrl', ['$scope', '$uibModalInstance', 'alertSummary',
+        function ($scope, $uibModalInstance, alertSummary) {
+            $scope.title = "Edit notes";
+            $scope.placeholder = "Leave notes here...";
+            $scope.error = false;
+            $scope.alertSummaryCopy = angular.copy(alertSummary);
+
+            $scope.saveChanges = function () {
```

Comment:
May as well make this an arrow function:

`function() {` => `() => {`

Same for everything below.

Reply:
You mean something like this?

$scope.saveChanges = () => {
                $scope.modifying = true;
                $scope.alertSummaryCopy.saveNotes().then(() => {
                    _.merge(alertSummary, $scope.alertSummaryCopy);
                    $scope.modifying = false;
                    $scope.error = false;

                    $uibModalInstance.close();
                }, () => {
                    $scope.error = true;
                    $scope.modifying = false;
                }
                );
            };`
`

Reply:
If you're referring to the _then_ callback param definition from line 82, I would agree.
If instead something like the code I presented above is required, I don't really see the need for using the arrow syntax. We don't reference any _this_ in our code, to gain any binding safety.
It kinda looks like an overuse and doesn't comply with the way the module was written.

Reply:
Arrow functions are now generally considered preferable in new javascript code:

http://airbnb.io/javascript/#arrow-functions

But there is an argument for being consistent with what's already there too (we can always do a bulk change later). I don't have a strong preference on this, so you can just leave it as-is if you prefer as I suggested earlier.